app/controllers/admin/dashboard_controller.rb

```diff
@@ -1,7 +1,8 @@
-class Admin::DashboardController < ApplicationController
+class Admin::DashboardController < Admin::BaseController
   before_action :require_admin
```

Comment:
move this to base controller

Reply:
Good catch! It was late enough last night that I thought I was just seeing some Rails magic automatically import this for me haha 🤣